pyanaconda/ui/tui/spokes/installation_progress.py

```diff
@@ -131,14 +131,22 @@ def show_all(self):
 
         util.ipmi_report(IPMI_FINISHED)
 
+        if self.instclass.eula_path is not None:
+            # Notify user about the EULA (if any).
+            print(_("Installation complete"))
+            print('')
+            print(_("Use of this product is subject to the license agreement found at:"))
+            print(self.instclass.eula_path)
+            print('')
+
         # kickstart install, continue automatically if reboot or shutdown selected
         if flags.automatedInstall and self.data.reboot.action in [KS_REBOOT, KS_SHUTDOWN]:
             # Just pretend like we got input, and our input doesn't care
             # what it gets, it just quits.
             raise ExitMainLoop()
 
     def prompt(self, args=None):
-        return Prompt(_("Installation complete. Press %s to quit") % Prompt.ENTER)
+        return Prompt(_("Press %s to quit") % Prompt.ENTER)
```

Comment:
It looks like users will not see the `Installation complete.` message, if there is no EULA. Is it ok?

Reply:
That looks like some sort of cherry-pick artifacts, I'll drop it.